Home/index.html

```diff
@@ -0,0 +1,160 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>E-com whitepaper</title>
+    <link rel="stylesheet" href="../Sass/style.css" />
+    <link
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.3.0/css/font-awesome.css"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link rel="stylesheet" href="../Sass/style.css" />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC"
+      crossorigin="anonymous"
+    />
+  </head>
+  <body>
+    <header>
+      <nav class="mobileNav d-md-none">
+        <i class="fa fa-bars pt-4 fa-3x" id="hamburger" aria-hidden="true"></i>
+        <img id="closeX" class="d-none" src="../img/close.png" alt="" />
+      </nav>
+      <nav class="desktopNav mt-3">
+        <div
+          class="destopNavItems d-flex align-items-center justify-content-between"
+        >
+          <div class="logo ms-5 d-none d-md-block">
+            <a class="logo" href="./index.html"
+              ><img src="../img/logo.jpeg" alt=""
+            /></a>
+          </div>
+          <ul
+            id="desktopList"
+            class="items list-unstyled d-flex gap-4 align-items-center d-none d-md-flex flex-column flex-md-row"
+          >
+            <li>
+              <a class="text-decoration-none text-dark" href="#program"
+                >Program</a
+              >
+            </li>
+            <li>
+              <a class="text-decoration-none text-dark" href="#speakers"
+                >Speakers</a
+              >
+            </li>
+            <li>
+              <a class="text-decoration-none text-dark" href="#sponsors"
+                >Sponsors</a
+              >
+            </li>
```

Comment:
- [ ] As I can see the nav bar here is not the same with the original design so kindly in the bigginng update the background color for the nav in the desktop version with the same margin and padding and don't forget to add the small navbar above the navbar to match the original design so please fix what I asked

- [ ] Kindly make the about link in the same place to be in the left side without backgrounf color to it so kindly fix that also to match the original design since it it is one of the requirements 💪🏻✅
- Your design
![image](https://user-images.githubusercontent.com/58553711/181145006-8142768b-d69e-4dd2-8a60-7ca6b10788f4.png)


- Original design
![image](https://user-images.githubusercontent.com/58553711/181145083-9acb94e3-290f-48a7-b98f-c2420173280c.png)

Reply:
about page repositioned